config/locales/en.yml

```diff
@@ -40,3 +40,11 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    coauthor_notification:
+      subject: "[%{app_name}] Co-Author Notification"
+      edit_the: "Edit the %{type}"
+      part1: "You have been listed as a coauthor on the following %{type}:"
```

Comment:
For some languages, the word for "following" will also change depending on the %{type}, just like "the" (as Cesy mentioned above, and also in line 50). I believe we'll need full strings for each different type.